lib/ansible/modules/user.py

```diff
@@ -1187,6 +1201,9 @@ def ssh_key_gen(self):
         if self.ssh_bits > 0:
             cmd.append('-b')
             cmd.append(self.ssh_bits)
+        if self.ssh_derivation_rounds > 0:
```

Comment:
is not None (default/unset)

Reply:
Is that correct, or needs to be changed.